webtau-core/src/main/java/com/twosigma/webtau/expectation/equality/CompareToComparator.java

```diff
@@ -44,126 +53,62 @@
     private final List<ActualPathMessage> missingMessages = new ArrayList<>();
     private final List<ActualPathMessage> extraMessages = new ArrayList<>();
 
-    private final boolean isNegative;
+    private AssertionMode assertionMode;
 
     public static CompareToComparator comparator() {
-        return new CompareToComparator(false);
-    }
-
-    public static CompareToComparator negativeComparator() {
-        return new CompareToComparator(true);
+        return new CompareToComparator(null);
     }
 
-    /**
-     * creates new instance of comparator without any collected mismatches. Preserves isNegative switch.
-     * @return fresh copy of equal comparator
-     */
-    public CompareToComparator freshCopy() {
-        return new CompareToComparator(isNegative);
+    public static CompareToComparator comparator(AssertionMode assertionMode) {
```

Comment:
Why did you choose static methods rather than c'tors?

Reply:
It made more sense when there were two different versions.

Reply:
OK.  We probably should change to c'tors at some point but not a biggie.